src/components/views/context_menus/TopLeftMenu.js

```diff
@@ -108,7 +108,7 @@ export class TopLeftMenu extends React.Component {
             </AccessibleButton>
         );
 
-        return <div className="mx_TopLeftMenu mx_HiddenFocusable" tabIndex={0} ref={this.props.containerRef}>
+        return <div className="mx_TopLeftMenu" ref={this.props.containerRef}>
```

Comment:
I guess tabIndex was removed on purpose here, to rather allow to navigate to elements inside the left panel?

Reply:
Yup